Persephone/Models/Preferences.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  Preferences.swift
+//  Persephone
+//
+//  Created by Daniel Barber on 2019/2/15.
+//  Copyright © 2019 Dan Barber. All rights reserved.
+//
+
+import Foundation
+
+struct Preferences {
+  var mpdHost: String? {
+    get {
+      return UserDefaults.standard.string(forKey: "mpdHost")
```

Comment:
What do you think about extracting the `UserDefaults` instance to an instance property?